messages/EventKeyTemplates.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
 		"telemetry": "This message is unused."
 	},
 	"warning": {
+		"pmd7IncompatibleRule": "PMD rule [%s] is incompatible with PMD 7: %s. If resolving these problems is non-viable for you, log an issue on github.com/forcedotcom/sfdx-scanner",
```

Comment:
This message hasn't been fully approved yet. Once the approved version comes in, I'll fix it.